qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -123,62 +124,130 @@ def calc_xdata(length_vector, length_multiplier):
     return np.array(xdata)
 
 
-def randomized_benchmarking_seq(nseeds=1, length_vector=None,
-                                rb_pattern=None,
-                                length_multiplier=1, seed_offset=0,
-                                align_cliffs=False,
-                                interleaved_gates=None,
-                                is_purity=False,
-                                group_gates=None):
-    """Get a generic randomized benchmarking sequence
+def randomized_benchmarking_seq(nseeds: int = 1,
+                                length_vector: Optional[List[int]] = None,
+                                rb_pattern: Optional[List[List[int]]] = None,
+                                length_multiplier: Optional[List[int]] = 1,
+                                seed_offset: int = 0,
+                                align_cliffs: bool = False,
+                                interleaved_gates: Optional[List[List[str]]] = None,
+                                is_purity: bool = False,
+                                group_gates: Optional[str] = None) -> \
+        (List[List[qiskit.QuantumCircuit]], List[List[int]],
+         Optional[List[List[qiskit.QuantumCircuit]]],
+         Optional[List[List[List[qiskit.QuantumCircuit]]]],
+         Optional[int]):
+    """Generate generic randomized benchmarking (RB) sequences.
 
     Args:
-        nseeds: number of seeds
-        length_vector: 'm' length vector of sequence lengths. Must be in
+        nseeds: The number of seeds. For each seed the function generates
+            a separate list of output RB circuits.
+
+        length_vector: Length vector of the RB sequence lengths. Must be in
             ascending order. RB sequences of increasing length grow on top of
             the previous sequences.
-        rb_pattern: A list of the form [[i,j],[k],...] which will make
-            simultaneous RB sequences where
-            Qi,Qj are a 2Q RB sequence and Qk is a 1Q sequence, etc.
-            E.g. [[0,3],[2],[1]] would create RB sequences that are
-            2Q for Q0/Q3, 1Q for Q1+Q2
-            The number of qubits is the sum of the entries.
-            For 'regular' RB the qubit_pattern is just [[0]],[[0,1]].
-        length_multiplier: if this is an array it scales each rb_sequence by
-            the multiplier
-        seed_offset: What to start the seeds at (e.g. if we
-            want to add more seeds later)
-        align_cliffs: If true adds a barrier across all qubits in rb_pattern
-            after each set of elements, not necessarily Cliffords
-            (note: aligns after each increment of elements including the
-            length multiplier so if the multiplier is [1,3] it will barrier
-            after 1 element for the first pattern and 3 for the second).
+
+            For example:
+
+            * ``length_vector = [1, 10, 20, 50, 75, 100, 125, 150, 175]``
+
+        rb_pattern: A list of the form ``[[i,j],[k],...]`` which will make
```

Comment:
It looks like the list must start with two qubits followed by one qubit

Reply:
OK